readme.md

```diff
@@ -53,10 +37,10 @@ A simple tweet processor for automatically and intelligently sorting tweets rela
 
 1. You can run the script manually:
 
-    `/path/to/python tweetsearch.py`
+    `poetry run python tweetsearch.py`
 
 2. Or you can run it on a schedule with a cron task such as (insert the following into your Cron file):
 
-    `*/2 * * * *	cd /path/to/home/github/spacex_smartweets && /path/to/python ./tweetsearch.py`
+    `*/2 * * * * cd /path/to/home/github/spacex_smartweets && /path/to/python ./tweetsearch.py`
```

Comment:
wouldn't this be `&& poetry run python tweetsearch.py` in the crontab?

Reply:
Yes you are right, have updated it in #6